### PR TITLE
use flake8 to check coding style at the root level

### PR DESCRIPTION
**Pull request by shajoezhu**

### CODE REVIEW

**Comment by jeromekelleher** [comment]
Hmmm. Not sure this is worth it on reflection. We have to make changes to `ez_setup.py`, which isn't ours. Can we exclude `ez_setup.py` from the checks?

File: .travis.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ before_script:
 # run_tests.py runs everything under the script: tag so only put commands
 # under it that we want to run (and want to be able to run) as local tests
 script: 
-  - flake8 tests ga4gh scripts
+  - flake8 . tests ga4gh scripts --exclude=ez_setup.py
```
Comment (shajoezhu) [inline .travis.yml]: flake8 check exclude ez_setup.py

**Comment by jeromekelleher** [comment]
Cool; +1.

**Comment by dcolligan** [comment]
I'm surprised that there are so few violations!

+1

**Pull request merged**